spec/imsc-hrm.html

```diff
@@ -230,7 +232,7 @@ <h2>Terms and Definitions</h2>
     </p>
     <p>
       A sequence of consecutive <dfn data-cite="ttml2#terms-intermediate-synchronic-document">Intermediate Synchronic Documents</dfn>
-      conforms to the Hypothetical Render Model if is processed without error as defined in <a
+      conforms to the Hypothetical Render Model if is processed without <a>error</a> as defined in <a
       href="#hypothetical-render-model-general"></a>.
```

Comment:
```suggestion
```

Reply:
I thought the link through to §6. General was helpful, so made an alternate change to address this suggestion instead. Does that work for you @palemieux ?

Reply:
I do not understand why the link is necessary since `<a>error</a>` already creates a link. I also think we need to avoid `i.e.`, which introduces potential ambiguities/double-meanings.